src/templates/article.js

```diff
@@ -92,17 +93,27 @@ const ArticleContentContainer = styled.section`
 `;
 
 export default function Article(props) {
+  const { article } = props.data;
+
   return (
     <Layout>
+      <Breadcrumb
+        paths={[
+          { url: '/', name: 'All categories' },
+          { url: `/${article.category.slug}/`, name: article.category.name },
+          { name: article.title },
+        ]}
+      />
+
       <article>
-        <ArticleTitle>{props.data?.article?.title}</ArticleTitle>
+        <ArticleTitle>{props.data.article.title}</ArticleTitle>
 
         <WhiteContainer>
           <ArticleContentContainer
             dangerouslySetInnerHTML={{
               __html: documentToHtmlString(
```

Comment:
Could we use documentToReactComponents here? https://www.npmjs.com/package/@contentful/rich-text-react-renderer